core/eval.go

```diff
@@ -49,6 +49,8 @@ func init() {
 	serverID = fmt.Sprintf("%s:%d", config.Host, config.Port)
 }
 
+const COUNT = "COUNT"
```

Comment:
nit: I guess we can avoid this, they are one and the same.

Reply:
it fails on linting and also the github actions failed beacuse of this

Reply:
I will look into QWATCH implementation and will try to integrate the same